XLS-0056d-batch/README.md

```diff
@@ -0,0 +1,535 @@
+
+### A.7: How is the `INDEPENDENT` mode any different than existing behavior with [tickets](https://xrpl.org/docs/concepts/accounts/tickets)?
+
+Tickets require temporarily having a reserve for all the tickets you want to create, but an `INDEPENDENT` mode `Batch` transaction doesn't, for the low cost of 10 extra drops.
+
+On the flip side, tickets are still needed for other use-cases, such as needing to coordinate multiple signers or needing out-of-order transactions.
+
+### A.8: Is it possible for inner transactions to end up in a different ledger than the outer transaction?
+
+No, because the inner transactions skip the transaction queue. They are already effectively processed by the queue via the outer transaction. Inner transactions will also be excluded from consensus for the same reason.
+
+### A.9: How does this work in conjunction with [XLS-75d](https://github.com/XRPLF/XRPL-Standards/discussions/218)? If I give an account powers over the `Batch` transaction, can it effectively run all transaction types?
+
+No, `OnBehalfOf` will not be allowed on `Batch` transactions. Instead, the inner transactions must include the `OnBehalfOf` field.
+
+### A.10: What if I want some error code types to be allowed to proceed, just as `tesSUCCESS` would, in e.g. an `ALLORNOTHING` case?
+
+This was deemed unnecessary. If you have a need for this, please provide example use-cases.
+
+### A.11: Why do the inner transactions need to include sequence numbers? 
+
+The primary use of sequence numbers is to prevent hash collisions - two otherwise-identical transactions must have different sequence numbers (or different `TicketSequence` values).
+
+In addition, some objects, such as [offers](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/offer/#offer-id-format) and [escrows](https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/escrow/#escrow-id-format), use the sequence number of the creation transaction as a part of their ledger entry ID generation, to ensure uniqueness of the IDs.
+
+For this reason, inner transactions must include sequence numbers.
+
+### A.12: What if I want the inner transaction accounts to handle their own fees?
+
+That is not supported. This also allows fee escalation to be calculated on the total cost of the transaction, instead of just on the overhead, and ensures that even if all of the transactions fail, the transaction fees are still charged.
+
+### A.13: How will [transaction simulation](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0069d-simulate) work with Batch?
+
+Some extra processing will be needed for that. As a result, Batch transactions likely won't be able to be simulated at first.
+
```

Comment:
Not sure if we want this here or in developer docs. 

1. Error Handling of Inner Batch Transactions is done using the debug stream. 
2. ParentBatchID is included on each of the Inner Transaction Metadata, when the result is tes or tec.

Reply:
Another point would be the cultivation when the inner transactions create and then consume an object and how the object ID is generated before the submission of the batch. Again, maybe developer docs is a better place for this.

Reply:
Same with AccountActivation, maybe just a note as it might be confusing to sign something for an account that doesnt yet exist.

Reply:
The object ID feels like a developer docs note, but I added the rest.